### PR TITLE
FIX warnings during doc gen and add all modules

### PR DESCRIPTION
**Pull request by morihladko**
This fixes all warnings during sphinx documentation generation and adds all modules to the API section except `zsl.tasks.zsl.cors_test_task` which can't be fixed easily. I created a separate task for it.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
           'celery': ['zsl_client'],
           'gearman': ['zsl_client', 'gearman'],
           'alembic': ['alembic'],
-          'documentation': ['sphinx', 'recommonmark', 'sphinx_rtd_theme']
+          'documentation': ['sphinx', 'recommonmark', 'sphinx_rtd_theme', 'alembic']
```
Comment (babky-atteq) [inline setup.py]: 👍

**Comment by morihladko** [comment]
No this should be it, although I tried to run in python 3 environment and it failed on not finding injector package, which is weird, I couldn't find a simple fix, but I guess it never run in python 3, so I created a separate issue for that.

File: setup.py (hunk 1 above)
Comment (babky-atteq) [inline setup.py]: not 80 chars per line

**Comment by babky-atteq** [comment]
For me this is approved, now it is up to you to check my changes.

**Comment by morihladko** [comment]
Yes it looks ok, but I'm not the reviewer, so you have to approve it.

**Comment by babky-atteq** [comment]
But I've made a major change, which should've been via a new branch, but, I hoped for a fast resolution, Thx.

**Pull request merged**